types/react-native-web/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "rules": {
+        "@definitelytyped/no-type-only-packages": "off",
+        "@typescript-eslint/no-invalid-void-type": "off",
+        "@typescript-eslint/no-empty-interface": "off"
```

Comment:
Rather than having these in the esline config, please just write an inline suppression on each interface you'd like to leave empty, so no (unintentional) new empty interfaces get added in the future. Likewise with the `void`s.

Reply:
Done 👌